docs/release-notes/index.md

```diff
@@ -1,5 +1,6 @@
 # Release Notes
 
+- [Percona Monitoring and Management 2.27.0](2.27.0.md)
```

Comment:
We need to fix this:
The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - release-notes/2.27.0.md
  - using/advisors/innodb_log_buffer_flushed_to_disk_after_each_trx.md
  
I do not see the 2.27.0.md in the navigation
<img width="487" alt="Screenshot 2022-04-12 at 12 58 01" src="https://user-images.githubusercontent.com/81549/162934574-74063647-f154-4298-8845-e854f5b09c2a.png">

Reply:
@atymchuk I haven't made changes in the mkdocs-base.yml yet. I will be raising a separate PR for that.